### PR TITLE
Icecoon's Weapons Pack upgrade

### LINKED ISSUE #18844: Getting an ammo error for 654 round upon game

### PR DESCRIPTION
**Pull request by BorkBorkGoesTheCode**
Ok, I have expanded the scope of this PR to include the rest of the ammo rebalance.

First, I relabeled the earlier reloaded 6.54x42mm cartridge and added a reloaded version. The wildcat 6.54x42mm has increased damage, at a cost of increased weight and low armor piercing ability.

The 6.54x42mm cartridges in general are now larger, heavier and less accurate, but they cause more damage.

Part four has been merged into this PR. The gun dispersion ratings have been increased. The SVS-24 series have 3-round bursts, All the guns have proper handling this time, to reflect their recoil compensators. And the guns have magazines.

### CODE REVIEW

File: data/mods/EW_Pack/ew_ammo.json [hunk 1]
```diff
@@ -72,16 +60,16 @@
         "name_plural" : ".20 DREAD Pellets",
         "symbol" : "=",
         "color" : "light_gray",
-        "description" : "These metal pellets are propelled with the help of electromagnets, therefore they do not produce sound, flash or heat signature.",
+        "description" : "Small .20 caliber ferromagnetic pellets. These pellets are propelled with the help of electromagnets; therefore they do not produce much sound, flash or heat signature.",
```
Comment (illi-kun) [inline data/mods/EW_Pack/ew_ammo.json]: Double space after period, please.

**Comment by BorkBorkGoesTheCode** [comment]
Ready.

**Comment by BorkBorkGoesTheCode** [comment]
Is anything wrong with this, from a design perspective?

**Comment by Rivet-the-Zombie** [comment]
Looks fine to me.

**Comment by BorkBorkGoesTheCode** [comment]
The bugfix for the itemgroup was PR'd in another PR ( #18871 ) in case this wasn't acceptable. If it get's merged, I can rebase the bugfix out of this one.

**Comment by BorkBorkGoesTheCode** [comment]
Removed bugfix.

**Comment by BorkBorkGoesTheCode** [comment]
Alright. The itemgroups are fixed.

**Comment by BorkBorkGoesTheCode** [comment]
@Coolthulhu @illi-kun @Rivet-the-Zombie  In general, this is ready.

**Comment by BorkBorkGoesTheCode** [comment]
@redice99 What do you think?

**Comment by BorkBorkGoesTheCode** [comment]
I volunteer as maintainer of Icecoon's Arsenal.

**Pull request closed without merging**